libraries/interop-rx2/src/main/kotlin/com/bumble/appyx/interop/rx2/connectable/NodeConnector.kt

```diff
@@ -31,7 +33,7 @@ class NodeConnector<Input, Output>(
     }
 
     override fun onCreate(lifecycle: Lifecycle) {
-        lifecycle.subscribe(onCreate = { flushOutputCache() })
+        lifecycle.subscribe(onCreate = { flushOutputCache() }, onDestroy = { intakeDisposable?.dispose() })
```

Comment:
Possibly we should log an error, or raise an exception if input/output is invoked after `onDestroy` is called, rather than just swallowing silently?

Reply:
sorry, I wasn't talking about lifecycles here @CherryPerry. I meant if someone called `input.accept`/`output.accept` after onDestroy for the node was called.

In this case the output event would be ignored, but the input event would not.

Reply:
IMHO do not see it as a problem that we should take care of, swallowing is OK for me.